src/test/scala/codacy/stylelint/ConfigTesting.scala

```diff
@@ -0,0 +1,171 @@
+package codacy.stylelint
+
+import better.files._
+import com.codacy.plugins.api.Source
+import play.api.libs.json._
+//import com.codacy.plugins.api.results.{Parameter, Pattern, Result, Tool}
+import com.codacy.plugins.api.results._
+import org.specs2.mutable.Specification
+
+import scala.util.Try
+
+class ConfigTesting extends Specification {
+
+  "This is a specification for the functioning of the generation/read of the configuration file for the stylelint tool. \n As such ".txt
+
+  "Stylelint" should {
+    "utilize the users configuration file when one is provided" in {
+      val (workspace,source) = createTemporaryWorkspace()
+
+      val expectedResult = Try(
```

Comment:
instead of this you should do the assertions using `result must beSuccessfulTry.withValues...`

[Take a look at specs2 documentation.](https://etorreborre.github.io/specs2/guide/SPECS2-3.6.6/org.specs2.guide.Matchers.html)